File: chapter-03/exercise-12-computing-simple-interest/ComputingSimpleInterest.java
import java.math.BigDecimal;
import java.math.RoundingMode;

class ComputingSimpleInterest {
  public static void main(String[] args) {
    var line = System.console().readLine("Enter the principal: ");
    var principal = Double.parseDouble(line);

    line = System.console().readLine("Enter the rate of interest: ");
    var rate = Double.parseDouble(line);

    line = System.console().readLine("Enter the number of years: ");
    var time = Integer.parseInt(line);

    var amount = new BigDecimal(principal * (1 + (rate / 100) * time)).setScale(2, RoundingMode.UP);

    var output = String.format("After %d years at %.2f%%, the investment will be worth $%s", time, rate, amount);
    System.out.println(output);
  }
}

File: 12-computing-simple-interest/computing_simple_interest.py
principal = float(input("Enter the principal: "))
rate = float(input("Enter the rate of interest: "))
years = int(input("Enter the number of years: "))

amount = round(principal * (1 + (rate / 100) * years), 2)

print(f"\nAfter {years} years at {rate}%, the investment will\n"
      f"be worth ${amount:.2f}")

